hack/patch_node.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -e
+
+# This script can be used in order to create/remove/update k8s extended resources named prow/sriov
+# see https://kubernetes.io/docs/tasks/administer-cluster/extended-resource-node for more details.
+# The resource will be created node wise, so each sriov node should be configured according
+# the desired capacity (according how many PFs it has, and how many jobs are actually desired to run simultaneously).
+# After creation, the resource would appear as allocatable on node's yaml after around 20 seconds.
+# The following command can be used in order to check it was created successfully.
+# timeout 60s bash -c "until oc get node $NODE -oyaml | grep prow/sriov | grep $CAPACITY | wc -l | grep 2; do sleep 1; done"
+
+# Usage:
+# ./patch_node.sh <NODE_NAME> <CAPACITY>
+
+PROXY_PORT=9999
+
+function finish {
+  rc=$?
+  if jobs 1 | grep -q Running; then
+    echo
+    echo "Killing proxy (pid=$proxy_pid)..."
```

Comment:
This global variable hurts :) Could we do something like `jobs -p | xargs kill` to kill all the background jobs in the current shell?

Reply:
I will verify that it doesn't touch jobs that were not created by the script, as if there are any,
the script shouldn't kill them.

Reply:
Indeed, I think the command above is safe, `jobs` only shows processes started in the current shell.

Reply:
Right, done thanks